.idea/runConfigurations/Start_packager_for_Integreat.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
```

Comment:
This starts the integreat-test-cms packager. Not sure if the naming is good therefore.

Reply:
I renamed them